Documentation/Books/Manual/Administration/Configuration/SSL.md

```diff
@@ -6,15 +6,15 @@ SSL Endpoints
 
 Given a hostname:
 
-`--server.endpoint tcp://hostname:port`
+`--server.endpoint http+ssl://hostname:port`
```

Comment:
I think it should be `tcp+ssl`.

The arangod help says:
`--server.endpoint <string...>                               endpoint for client requests (e.g. 'http+tcp://127.0.0.1:8529', or 'vst+ssl://192.168.1.1:8529') (default: )`

I would really prefer `http / https / vst / vsts` but for some reason we have that syntax. Maybe other variations work as well but then we would need to update the help and the manpages.

Reply:
devel feels a bit broken:
- `http+ssl` and `tcp` work
- `tcp+ssl` and `http` do not work

Reply:
It should just `ssl://`.